<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpDialog</class>
 <widget class="QDialog" name="HelpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TreeMk User Guide</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="navigationLayout">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <widget class="QPushButton" name="homeButton">
       <property name="text">
        <string>Home</string>
       </property>
       <property name="toolTip">
        <string>Return to main help page</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
       <property name="toolTip">
        <string>Go to previous page</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="forwardButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Forward</string>
       </property>
       <property name="toolTip">
        <string>Go to next page</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="breadcrumbLabel">
       <property name="text">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">font-weight: bold; padding: 0 8px;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchLineEdit">
       <property name="placeholderText">
        <string>Search help topics...</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
       <property name="minimumWidth">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <widget class="QStackedWidget" name="leftPanelStack">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tocPage">
       <layout class="QVBoxLayout" name="tocLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QListWidget" name="tableOfContents"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="searchResultsPage">
       <layout class="QVBoxLayout" name="searchResultsLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="searchResultsLabel">
          <property name="text">
           <string>Search Results</string>
          </property>
          <property name="styleSheet">
           <string notr="true">font-weight: bold; padding: 8px; background-color: palette(mid);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="searchResultsList"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="contentViewContainer" native="true">
      <layout class="QVBoxLayout" name="contentViewLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="printButton">
       <property name="text">
        <string>Print...</string>
       </property>
       <property name="toolTip">
        <string>Print current help page</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>HelpDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>949</x>
     <y>669</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
